packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -1523,6 +1524,31 @@ integTest(
   }),
 );
 
+integTest('cdk diff --import-existing-resources print diff of imported resources', withDefaultFixture(async (fixture) => {
+  const repositoryName = `${fixture.stackNamePrefix}-repository`;
+  await fixture.aws.ecr.send(new CreateRepositoryCommand({ repositoryName: repositoryName }));
+
+  try {
+    await fixture.cdkDeploy('simplified-import', {
+    });
```

Comment:
The integ-test steps:

1. Create an AWS resource (e.g. ECR repository) to be imported 
2. Deploy a stack without importing resource (because cdk diff only works for an existing stack)
3. Run cdk diff with the stack template that contains the resource to be imported
4. assert cdk diff output
5. destroy stack
6. delete the AWS resource created at step 1